emission/tests/pipelineTests/TestExpectationPipeline.py

```diff
@@ -100,13 +110,17 @@ def testProcessedAgainstRaw(self):
             self.assertIn("expectation", trip["data"])
             raw_expectation = eace.get_expectation(trip)
             if raw_expectation["type"] == "none":
-                self.assertEqual(trip["data"]["expectation"], {"to_label": False})
+                self.assertEqual(trip["data"]["expectation"]["to_label"], False)
             elif raw_expectation["type"] == "all":
-                self.assertEqual(trip["data"]["expectation"], {"to_label": True})
+                self.assertEqual(trip["data"]["expectation"]["to_label"], True)
             else:
                 print("Expectation behavior for "+str(raw_expectation)+" has not been implemented yet; not testing. Value is "+str(trip["data"]["expectation"]))
             # TODO: implement tests for the other configurable expectation types once they've been implemented
 
+    def testConfidenceThreshold(self):
+        for trip in self.expected_trips:
+            self.assertTrue("confidence_threshold" in trip["data"])
```

Comment:
Shouldn't you have a tighter check than this? You know what the `confidence_threshold` should be, why not check for it directly?

Reply:
I would like to see some actual values even if they are all the same